src/include/parser/expression/expression_util.h

```diff
@@ -1,78 +0,0 @@
-#pragma once
-
-#include <algorithm>
-#include <cstdlib>
-#include <set>
-#include <string>
-#include <vector>
-
-#include "catalog/catalog_defs.h"
-#include "common/managed_pointer.h"
-#include "parser/expression/abstract_expression.h"
-#include "parser/expression/aggregate_expression.h"
-#include "parser/expression/column_value_expression.h"
-
-namespace terrier::parser::expression {
-
-/**
- * An utility class for Expression objects
- */
-class ExpressionUtil {
```

Comment:
Was this code moved somewhere else?

Reply:
The code was merged into `src/include/parser/expression_util.h`.